addons/website_event/static/src/scss/event_templates_common.scss

```diff
@@ -76,6 +81,13 @@ $o-wevent-color: $body-color;
             box-shadow: 0 0 0 0.25rem rgba(134, 102, 126, 0.5);
         }
     }
+
+    @media not all and (hover: none) {
```

Comment:
You're right, we likely don't need the @media not all and (hover:none) query if the goal is to always keep the 'X' (cancel button) visible on both desktop and mobile. We can simply use the input[type="search"] selector to set the opacity directly, ensuring it's always visible?

Reply:
Arf. I think I would just skip that point from the spec:

I noticed that `::-webkit-search-cancel-button` is not standard (https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-search-cancel-button). So, even if we find a solution, it won't work on all browsers. For that reason, I would not spend too much time on it. I think it's okay if we skip that point given that it's a really minor change and we have time constraints :smiley:

Reply:
Yes, you're right. I've messaged Awa about this, mentioning that the issue won't work across different browsers since it's tied to the -webkit- pseudo-element. However, since we only get the cross icon in Chrome, maybe we could keep the change to ensure it works there?

Reply:
Yes, we can do the best effort. If you find a simple solution, we can go for it!

Thank you for your followup. I will green the task as the rest of the code seems good to me :smile:

TODO list:
- [ ] Discuss the folding strategy to use with the PO/reviewer.
- [ ] Check what we do with the cross icon.